### PR TITLE
move translations for commons-ui

### PR DESCRIPTION
**Pull request by EtienneLt**
linked to https://github.com/gridsuite/commons-ui/pull/526

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -160,14 +160,25 @@
...
   "seriesResistance": "Series resistance (Ω)",
+  "seriesResistance1": "Series resistance 1 (Ω)",
+  "seriesResistance2": "Series resistance 2 (Ω)",
+  "seriesResistance3": "Series resistance 3 (Ω)",
   "seriesReactance": "Series reactance (Ω)",
+  "seriesReactance1": "Series reactance 1 (Ω)",
+  "seriesReactance2": "Series reactance 2 (Ω)",
+  "seriesReactance3": "Series reactance 3 (Ω)",
+  "shuntConductance": "Shunt conductance (μS)",
+  "shuntSusceptance": "Shunt susceptance (μS)",
```
Comment (thangqp) [inline src/translations/en.json]: Is this in the scope of expert filter for hdvc line?

Comment (EtienneLt) [reply src/translations/en.json]: yes it is not needed for hvdc it is for 3 windings tranformers I'll change the links

Comment (thangqp) [reply src/translations/en.json]: In my opinion, it is better to not add more translate keys in explorer-app, we can profit this PR to clean old keys which similar to new keys, so we can clean : 
voltageLevelIdXXX, 
terminalXXXConnected, 
countryXXX, 
seriesResistance, 
seriesReactance

Comment (EtienneLt) [reply src/translations/en.json]: So I put the new translations in commons ui ?

Comment (thangqp) [reply src/translations/en.json]: YES, keep this PR only for cleaning old keys after merge PR commons-ui

Comment (EtienneLt) [reply src/translations/en.json]: alright

**Review by thangqp — APPROVED** [review]

**Pull request merged**